scripts/west_commands/runners/nrfjprog.py

```diff
@@ -104,10 +105,10 @@ def do_run(self, command, **kwargs):
                 program_cmd
             ])
         else:
-            if self.family == 'NRF51':
-                commands.append(program_cmd + ['--sectorerase'])
-            else:
+            if self.family == 'NRF52':
                 commands.append(program_cmd + ['--sectoranduicrerase'])
+            else:
```

Comment:
Hm... this should break tests cases and I'm not sure why it's not.

```
    '9YNN':
    (['nrfjprog', '--program', RC_KERNEL_HEX, '-f', 'NRF91', '--snr', TEST_DEF_SNR, '--sectoranduicrerase'],  # noqa: E501
     ['nrfjprog', '--reset', '-f', 'NRF91', '--snr', TEST_DEF_SNR]),
```

Reply:
This is now fixed, thanks for catching it!